src/main/java/io/kestra/plugin/shopify/customers/Get.java

```diff
@@ -0,0 +1,100 @@
+package io.kestra.plugin.shopify.customers;
+
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.runners.RunContext;
+import io.kestra.core.serializers.JacksonMapper;
+import io.kestra.plugin.shopify.AbstractShopifyTask;
+import io.kestra.plugin.shopify.models.Customer;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import jakarta.validation.constraints.NotNull;
+import java.net.URI;
+import java.net.http.HttpClient;
```

Comment:
Use `io.kestra.core.http.client.HttpClient` everywhere instead of `java.net.http.HttpClient`